dpu-cni/pkgs/cniserver/cniserver_test.go

```diff
@@ -80,11 +85,26 @@ var _ = g.Describe("Cniserver", func() {
 				CNIVersion: cniVersion,
 			}
 			g.It("should get a correct response from the post request", func() {
-				resp, ver, err := plugin.PostRequest(PrepArgs(cniVersion))
+				resp, ver, err := plugin.PostRequest(PrepArgs(cniVersion, "ADD"))
```

Comment:
(Optional) I know it is horrible on its on. But can we have a const added something for ADD and DEL? like "cnitypes.CommandAdd"